src/core/Tooltip/style.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from "@emotion/css";
+import {
+  AppThemeOptions,
+  fontBodyXs,
+  getColors,
+  getShadows,
+  getSpacings,
+} from "../styles";
+
+export const tooltipCss = (theme: AppThemeOptions): string => {
+  const props = { theme };
+  const colors = getColors(props);
+  const shadows = getShadows(props);
+  const spacings = getSpacings(props);
+
+  return css`
+    ${fontBodyXs(props)}
```

Comment:
Here we also need to call `fontBodyXs()` with `props`, because `css()` from `@emotion/css` doesn't call `fontBodyXs()` with props like `@emotion/styled` does